File: leetcode/largestelement.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    int largestElement(vector<int>& nums) {
        int highest=nums[0];
        for (int i=1;i<nums.size();i++){
            if (nums[i]>highest){
                highest=nums[i];
            }
        
        }
        return highest;
    }
};

int main(){

	
	Solution s1;
	Solution s2;
	vector<int> arr1={2,7,1,0};
	vector<int> arr2={-50,10,-40,30};
	cout<<"Largest element in array is"<<s1.largestElement(arr1);
	cout<<"Largest element in array is"<<s2.largestElement(arr2);
	return 0;
}

File: leetcode/largestelement.py
def findLargest(arr1):
    largest=arr1[0]
    for i in range(1, len(arr1)):
        if largest<arr1[i]:
            largest=arr1[i]
    return largest

if __name__=="__main__":
    arr1=[2,7,3,1]
    print("largest element in the array is", findLargest(arr1))
